index.bs

```diff
@@ -991,6 +1129,34 @@ partial interface ModelBuilder {
     </div>
 </div>
 
+### resample ### {#api-modelbuilder-resample}
+Resample the tensor values from the source to the destination dimensions according to the scaling factors.
+<script type=idl>
+enum InterpolationMode {
+  "nearest-neighbor",
+  "linear"
+};
+
+dictionary ResampleOptions {
+  InterpolationMode mode = "nearest-neighbor";
+  sequence<float> scales;
```

Comment:
I am not sure whether it needs to allow specifying the new size. With that developers would be able to set either scales or size for when working with different frameworks.

Reply:
I think the target sizes can be easily converted to `scales`. It could be redundant to do both.

Reply:
I suppose the usage of target sizes is similar to the `outputSizes` of transposed conv2d. It is used when the input size is unknown when graph building time.

Reply:
Good point. The input to `resample` can indeed be a free dimension input to the model. I added the `sizes` option as an alternative to `scales` as suggested.

Reply:
thanks, looks good now.